ipi/engine/properties.py

```diff
@@ -2613,6 +2686,21 @@ def get_ti_term(self, atom=""):
 
         return ti
 
+    def get_exchange_distinct_prob(self):
+        if not self.nm.exchange:
```

Comment:
I'd throw and error if there are no bosons (and formulate this as `if self.nm.exchange is None` which is more explicit.